templates/appliance/plex.html

```diff
@@ -0,0 +1,163 @@
+{% extends "appliance/_base-appliance.html" %}
+
+{% block title %}Plex Media Server | Ubuntu Appliance{% endblock %}
+
+{% block meta_description %}Plex magically organizes your media libraries and streams them to any device – including all your video, music, and photo libraries.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1zgOl4GCK6O6Clv6oxp-DXDSKYIKO2pvpXaYXALiTlpo/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--light is-shallow">
+  <div class="row">
+    <div class="col-8">
+      <div class="p-media-object--large u-no-margin--bottom">
+        <img src="https://assets.ubuntu.com/v1/402069bd-plex-media-server.png?w=90" class="p-media-object__image is-round" height="90" width="90">
+        <div class="p-media-object__details">
+          <h1 class="p-heading--two">
+            Plex Media Server
+          </h1>
+          <p class="p-heading--four">Ubuntu Appliance</p>
+          <ul class="p-inline-list--middot u-no-margin--bottom">
+            <li class="p-inline-list__item">By Plex, Inc. (plexinc)</li>
+            <li class="p-inline-list__item">Entertainment</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="col-4 u-align--right">
+      <form class="p-form--inline" action="#">
+        <div class="p-form__group">
+          <label for="platform" class="u-off-screen">Platform</label>
+          <select id="platform">
+            <option value="rpi">Raspberry Pi</option>
+            <option value="kvm">KVM</option>
+            <option value="intel-nuc">Intel NUC</option>
+          </select>
+        </div>
+        <div class="p-form__group">
+          <submit href="/appliance/plex/rpi" class="p-button--positive p-form__control">Download</submit>
```

Comment:
Never seen this before. Do you mean `<button type="submit"...>`

Reply:
lol, yes. Karl's brain: "I'll just change that to a submit button"